server/routes/journeys/referral/on-behalf-of/tests.cy.ts

```diff
@@ -0,0 +1,50 @@
+context('Make a Referral Journey', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubGetPrisoner')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubComponents')
+    cy.task('stubAreaOfWork')
+    cy.task('stubIncidentLocation')
+    cy.task('stubIncidentType')
+    cy.task('stubIncidentInvolvement')
+    cy.task('stubContribFactors')
```

Comment:
if it's in java mockito you'd get a `UnnecessaryStubbingException`

i try not to stub more than necessary, but am okay with copy and pasting the same stubs for all tests in a journey.

Reply:
Ah you're right :) changed